### PR TITLE
Draft russian localization.

### PR DESCRIPTION
**Pull request by doktorjet**
Draft russian localization.
Read a couple of comments in text.

### CODE REVIEW

File: GameData/NearFutureSpacecraft/Localization/ru.cfg [hunk 1]
```diff
@@ -0,0 +1,150 @@
+﻿// Draft russian localization. No tags.
+// © Dr. Jet
+
+// As tags mostly mimic Squad ones - it may be wize to remove them from localization files and write part config lines like this: 
+// tags = #autoLOC_500291 nearfuture orbital
```
Comment (ChrisAdderley) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: That's an absolutely hideous amount of work to go back and do that for everything. If you want to volunteer and validate though, be my guest

File: GameData/NearFutureSpacecraft/Localization/ru.cfg [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    #LOC_NFSpacecraft_monoprop-tank-375-1_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-375-2_title = FL-R-C3000 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-375-2_description = Бак больше-среднего-размера вмещает топлива больше-среднего-объёма. В данном случае - монотоплива.
+    #LOC_NFSpacecraft_monoprop-tank-375-2_tags = fueltank mono monopropellant propellant nearfuture
+    #LOC_NFSpacecraft_monoprop-tank-375-3_title = FL-R-C1500 Монотопливный бак
+    #LOC_NFSpacecraft_monoprop-tank-375-3_description = Строите большой технический отсек? Вам нужен большой монотопливный бак? Это то, что вам нужно.
+    #LOC_NFSpacecraft_monoprop-tank-375-3_tags = fueltank mono monopropellant propellant nearfuture
+
+    // Structural
+    #LOC_NFSpacecraft_engine-pod-1_title = SD-01 Гондола двигателя радиальной установки		// Why do you use aerocraft terms for spacecraft parts?
```
Comment (ChrisAdderley) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: How is "engine pod" an aerocraft term?

File: GameData/NearFutureSpacecraft/Localization/ru.cfg [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    #LOC_NFSpacecraft_rcsblock-orbital-5way-1_title = RX-55 Улучшенный блок двигателей РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-5way-1_description = Серия блоков двигателей RX от «БАЛАНС Инжиниринг» жертвует эффективностью в атмосфере ради высокой эффективности в космосе. This block is a jack of all trades with five nozzles, allowing thrust in five directions.
+    #LOC_NFSpacecraft_rcsblock-orbital-5way-1_tags = cluster control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
+    #LOC_NFSpacecraft_rcsblock-orbital-linear-1_title = RX-1 Линейный двигатель РСУ
+    #LOC_NFSpacecraft_rcsblock-orbital-linear-1_description = Одиночное сопло серии RX обеспечивает тягу в одном направлении. Прикрепите несколько, ну, и мир - ваш неопознанный моллюск.		// Literal translation. I can't neither understand nor adapt/localize such weird humor... if it IS a humor in the first place... =o_O=
+    #LOC_NFSpacecraft_rcsblock-orbital-linear-1_tags = control dock maneuver manoeuvre react rendezvous rotate stab steer translate orion nearfuture
```
Comment (ChrisAdderley) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: Yeah colloquialism may not translate, feel free to remove it.

File: GameData/NearFutureSpacecraft/Localization/ru.cfg (hunk 2 above)
Comment (doktorjet) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: Engine pod strictly translates as "гондола двигателя". It's a thing, usually hosting a jet or piston engine on airplane or zeppelin.

**Egine pod** 
_noun
aeronautics_
an aircraft turbojet unit comprising the engine and its cowling suspended by a pylon, often below the wing
Collins English Dictionary. Copyright © HarperCollins Publishers

Pics:
https://www.google.ru/search?q=engine+pod&tbm=isch&tbo=u&source=univ&sa=X

File: GameData/NearFutureSpacecraft/Localization/ru.cfg (hunk 3 above)
Comment (doktorjet) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: So... what's the initial meaning? I can't find the comparable russian colloquialism if i don't understand it.

File: GameData/NearFutureSpacecraft/Localization/ru.cfg (hunk 2 above)
Comment (ChrisAdderley) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: Alright, you're free to disagree then... change the russian one to whatever you like.

File: GameData/NearFutureSpacecraft/Localization/ru.cfg (hunk 3 above)
Comment (ChrisAdderley) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: the world's your oyster? https://en.wiktionary.org/wiki/the_world_is_one%27s_oyster

Comment (doktorjet) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: Hm-m-m... that Shaekspir's phrase was never translated literally. "Everything is in my hands" is most common translation. No wonder I couldn't find a clue.

File: GameData/NearFutureSpacecraft/Localization/ru.cfg (hunk 2 above)
Comment (doktorjet) [inline GameData/NearFutureSpacecraft/Localization/ru.cfg]: Maybe "engine casing" or "engine cowling" (кожух двигателя)? I mean that part does not even had a pylon to be called "engine pod".

**Comment by ChrisAdderley** [comment]
Merged, thanks!

**Pull request merged**